src/cascadia/TerminalCore/Terminal.cpp

```diff
@@ -406,6 +406,21 @@ bool Terminal::IsTrackingMouseInput() const noexcept
     return _terminalInput->IsTrackingMouseInput();
 }
 
+// Method Description:
+// - If the clicked text is a hyperlink, open it
+// Arguments:
+// - The position of the clicked text
+std::wstring Terminal::GetHyperlink(const COORD position)
+{
+    auto attr = _buffer->GetCellDataAt(_ConvertToBufferCell(position))->TextAttr();

```

Comment:
Does this still work when the hyperlink has been scrolled off the screen? We're taking into the viewport  offset right?